tools/tf2openapi/types/constants.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+const (
```

Comment:
Unless these constants are used widely, consider just putting them in the file where they're used.